### PR TITLE
Reversed IsEngineOn, GetSiren and GetBeaconLightsOn

### PR DESCRIPTION
**Pull request by Segfaultd**
PR closes https://github.com/MafiaHub/MafiaMP/issues/73

Won't address the TODOs for now so lets keep it like this

### CODE REVIEW

**Review by Deewarz — APPROVED** [review]
You can remove these TODOs then : 
- https://github.com/MafiaHub/MafiaMP/blob/ca5b5de40b6438008f10e6780302d290fc90c5ff/code/server/src/core/builtins/vehicle.h#L242
- https://github.com/MafiaHub/MafiaMP/blob/ca5b5de40b6438008f10e6780302d290fc90c5ff/code/server/src/core/builtins/vehicle.h#L253

File: code/client/src/core/ui/vehicle_debug.cpp [hunk 1]
```diff
@@ -20,6 +20,8 @@ namespace MafiaMP::Core::UI {
 
         if (currentCar) {
             auto currentVehicle = currentCar->GetVehicle();
+            ImGui::Text("Car Ptr: %p", currentCar);
+            ImGui::Text("Vehicle Ptr: %p", currentVehicle);
```
Comment (Deewarz) [inline code/client/src/core/ui/vehicle_debug.cpp:24]: These are already at the bottom : https://github.com/MafiaHub/MafiaMP/pull/90/files#diff-9b678781c9bd580b6852983396c0fa61aee6c99734fc0e99a0f9c8a5f1419d97R178-R179

File: code/client/src/core/ui/vehicle_debug.cpp [hunk 2]
```diff
@@ -100,6 +102,11 @@ namespace MafiaMP::Core::UI {
                 currentVehicle->SetBeaconLightsOn(beaconsLight);
             }
 
+            bool isEngineOn = currentCar->IsEngineOn();
+            if (ImGui::Checkbox("Engine", &isEngineOn)) {
+                currentVehicle->SetEngineOn(isEngineOn, isEngineOn);
+            }
```
Comment (Deewarz) [inline code/client/src/core/ui/vehicle_debug.cpp:108]: Just to be sure, getter is on Car and setter on Vehicle?

Comment (Segfaultd) [reply code/client/src/core/ui/vehicle_debug.cpp:108]: exactly

**Pull request merged**